global-script.lua

```diff
@@ -2817,9 +2817,10 @@ function setupPlayerArea(params)
     obj.setVar("playerColor", color)  -- May be nil
     if color then
         obj.createButton({
-            label="Move to here", click_function="onSwapButtonClicked", function_owner=Global,
+            label="Swap with " .. color, click_function="onSwapButtonClicked", function_owner=Global,
```

Comment:
weren't you saying how putting the color into the label wasn't going to work?

Reply:
I can't have it say "Move Blue to this seat", because it'd need to show *your* color here, which is different for all six people.

I can have it say "Swap with Blue", since whether you're Green or Red or Orange or whatever, you're swapping with where Blue is.

Reply:
ahh whoops got things jumbled up in my head